components/Examples.vue

```diff
@@ -56,4 +60,10 @@ import zhCn from "element-plus/es/locale/lang/zh-cn";
 const timeValue = ref("");
 const hello = () => ElMessage.info("hello world");
 const helloSuccess = () => ElMessage.success("hello world");
+
+const color = useColorMode();
+const colorMode = computed({
```

Comment:
Maybe we can refer it.

https://github.com/antfu/vitesse-nuxt3/blob/main/components/DarkToggle.vue

Reply:
This example uses button+function , But i use switch+computed. Do you mean to change to button+function?

Reply:
Using switch is ok.

Reply:
Using switch is ok.